### DISCUSSION TITLE
Show Z-Wave JS Server version next to other version numbers

### DISCUSSION THREAD

**Discussion by frenck** [zwave-js/zwave-js-ui#426] [Ideas]
It would be nice if the version info listed here:

![image](https://user-images.githubusercontent.com/195327/106620716-210b5500-6572-11eb-8ee2-19e67d3e5b45.png)

Would also have the Z-Wave JS Server version shown.

**Comment by robertsLando** [comment]
Is it provided by zwavejs server?

**Reply by frenck** [reply]
I dunno. I'm just dropping an idea. I'm not aware of the internal of the Z-Wave JS server.

**Reply by balloob** [reply]
https://github.com/zwave-js/zwave-js-server/pull/124
